base/operators.jl

```diff
@@ -1035,11 +1035,25 @@ end
 ∘(f, g, h...) = ∘(f ∘ g, h...)
 
 function show(io::IO, c::ComposedFunction)
-    show(io, c.outer)
+    _showcomposed(io, c.outer)
```

Comment:
```suggestion
    c.outer isa ComposedFunction ? show(io, c.outer) : _showcomposed(io, c.outer)
```
This way `f ∘ g ∘ h` will print as `"f ∘ g ∘ h"` and not `"(f ∘ g) ∘ h"`